Python/sysmodule.c

```diff
@@ -2290,6 +2290,23 @@ sys__getframemodulename_impl(PyObject *module, int depth)
     return Py_NewRef(r);
 }
 
+/*[clinic input]
+sys._get_cpu_count_config
+Private function to get PyConfig.cpu_count is overridden or not
+[clinic start generated code]*/
+
+static PyObject *
+sys__get_cpu_count_config_impl(PyObject *module)
```

Comment:
@gpshead 

I wrote the comment about the need of magic string in https://github.com/python/cpython/pull/109667#discussion_r1343351179

PTAL

Reply:
Agreement with https://github.com/python/cpython/pull/109667#discussion_r1343372035